### DISCUSSION TITLE
Why the tags from TickTick are converted to lower case during sync?

### DISCUSSION THREAD

**Discussion by ycwu1030** [thesamim/TickTickSync#94] [Q&A] [answered]
Is it possible to keep the tag as it is when sync between obsidian and TickTick? Thanks!

**Comment by thesamim** [comment, marked as answer]
> Is it possible to keep the tag as it is when sync between obsidian and TickTick? Thanks!

At some point, Either Obsidian or TickTick only handled lower case tags. This seems to have changed. I will dig into it and make changes because apparently they now both have no issue with mixed case tags.

**Reply by ycwu1030** [reply]
Thanks! That would be helpful, there are lots of mixed case tags in both Obsidian and TickTick.

**Comment by thesamim** [comment]
Sorry, TickTick converts mixed case Tags to lower case from the API. Nothing I can do about it.

**Reply by ycwu1030** [reply]
Thanks for the info. May I know whether the ALL-Capital tags will also be converted to lower case or just mixed case? Thanks

**Reply by thesamim** [reply]
Just testing the API to answer that question, I am finding that handling of case is inconsistent. Just using postman to exercise the API (ie: no plugin code involved at all.) I have seen case preserved and  case changed. I haven't figured out the pattern yet.

I'm in the middle of another project right now. When I have more time, I will investigate further. 

Thanks in advance for your patience with this.

**Comment by thesamim** [comment]
Re-Opening pending further investigation.

**Comment by thesamim** [comment]
Forgot to update this discussion. #298 took care of that issue. Please try again.
